docs/data/routes/docs/nextjs/experience-editor/walkthrough/en.md

```diff
@@ -3,4 +3,109 @@ name: walkthrough
 routeTemplate: ./data/component-templates/article.yml
 title: Connect to the Experience Editor
 ---
-# Walkthrough: Connect to the Experience Editor
\ No newline at end of file
+# Walkthrough: Connect your Next.js app to the Experience Editor
+
+A Next.js application utilizes the [JSS Http rendering engine](/docs/fundamentals/services/view-engine) feature for integration to the Sitecore Experience Editor. This article will guide you through the required configuration.
```

Comment:
Please add a note at the top to the extent of,

> The `dotnet new` template for Next.js includes pre-configured integration with the Experience Editor.

Reply:
Added.